src/client/SessionPool.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022 vesoft inc. All rights reserved.
+//
+// This source code is licensed under Apache 2.0 License.
+
+#include "nebula/client/SessionPool.h"
+
+#include <folly/json.h>
+
+#include "common/time/TimeConversion.h"
+#include "nebula/client/ConnectionPool.h"
+
+namespace nebula {
+
+bool SessionPool::init() {
+  Config conf;
+  conf.maxConnectionPoolSize_ = config_.maxSize_;
+  conf.minConnectionPoolSize_ = config_.minSize_;
+  conf.idleTime_ = config_.idleTime_;
+  conf.timeout_ = config_.timeout_;
+  pool_->init(config_.addrs_, conf);
+  if (config_.spaceName_.empty()) {
+    return false;
+  }
+  if (config_.username_.empty() || config_.password_.empty()) {
+    return false;
+  }
+  std::string useSpace = "USE " + config_.spaceName_;
+  for (std::size_t i = 0; i < config_.maxSize_; ++i) {
+    // use space
```

Comment:
i < config_.minSize_

Reply:
Emm. It's a legacy of nebula-cpp. I will fix in later.